__tests__/app.test.js

```diff
@@ -28,15 +28,59 @@ describe("API Testing", () => {
     });
   });
 
-  describe.only("Ticket 4: GET /api/articles", () => {
+  describe("Ticket 4: GET /api/articles", () => {
     test("returns an array of articles which is sorted by date in descending order ", () => {
       return request(app)
         .get("/api/articles")
         .expect(200)
+        .then(({ body }) => {
+          const objOfArr = body.articles;
+          expect(objOfArr.length).toBe(12);
+        });
```

Comment:
on line 32, your test mentions sorting, so you'll need to use jest-sorted to check it's sorted as you expect